src/lib/ngsi/ContextAttributeVector.cpp

```diff
@@ -705,7 +705,7 @@ void ContextAttributeVector::applyUpdateOperators(void)
             break;
 
           default:
-            LM_E(("Runtime Error (unknown attribute type: %d)", upOp->valueType));
+            LM_E(("Runtime Error (unknown attribute type: %d on attribute %s)", upOp->valueType, vec[ix]->name.c_str()));
```

Comment:
```suggestion
            LM_E(("Runtime Error (unknown attribute value type: %d on attribute %s)", upOp->valueType, vec[ix]->name.c_str()));
```

To homogenize all these similar LM_E traces.